packages/biocaml/biocaml.dev/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+synopsis: "The OCaml Bioinformatics Library"
+description: """
+Biocaml aims to be a high-performance user-friendly library for
+Bioinformatics. See the [website](http://biocaml.org) for detailed API
+documentation. We welcome contributors and feedback from users. Please
+contact us with any comments and suggestions for features you would
+like added.
+"""
+maintainer: "biocaml@googlegroups.com"
+homepage: "http://biocaml.org"
+bug-reports: "https://github.com/biocaml/biocaml/issues"
+license: "LGPL + linking exception"
+dev-repo: "git+https://github.com/biocaml/biocaml.git"
+authors: [
+  "Sebastien Mondet"
+  "Ashish Agarwal"
+  "Philippe Veber"
+  "Christophe Troestler"
+  "Sergei Lebedev"
+  "David Koppstein"
+  "Alexander Gryzlov"
+  "Francois Berenger"
+]
+
+build: ["dune" "build" "-p" name "-j" jobs]
+
+depends: [
+  "base64"
+  "ocamlfind" {build}
```

Comment:
I could not find reference to ocamlfind, except for an omake (I think) file from 3 years ago in `etc`. Is it actually used?